doc-coverage/raco.rkt

```diff
@@ -0,0 +1,75 @@
+#lang racket
+
+(require raco/command-name
+         racket/cmdline
+         rackunit/docs-complete
+         "main.rkt")
+
+(module+ main
+
+  (define binding (make-parameter #f))
+  (define ratio (make-parameter #f))
+  (define ignore (make-parameter #f))
+  (define error-on-exit? (make-parameter #f))
+
+  (define args
+    (command-line
+     #:program (short-program+command-name)
+     #:once-any
+     [("-b" "--binding") b
+      "Check the documentation for a specific binding"
+      (binding (string->symbol b))]
+     [("-r" "--ratio") r
+      "Specify required documentation ratio"
+      (ratio (string->number r))]
+     [("-s" "--skip") s
+      "Specify regex of bindings to ignore"
+      (ignore (regexp s))]
+     #:args (file . files)
+     (cons file files)))
+
+  (for ([a (in-list args)])
+    (let/ec break
+
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (set! a (string->symbol a)))])
+        (namespace-require a))
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (fprintf (current-error-port) "Module ~a can not be loaded~n" a)
+                                   (error-on-exit? #t)
+                                   (break))])
```

Comment:
Since this is the only place the escape continuation is used, and it's only used to determine whether to go on to the `cond` body, I think it would read better if this namespace-require stuff was moved into some `try-namespace-require!` function that returned a boolean indicating success. Then this could be rearranged into something like:

```racket
(if (try-namespace-require! a)
  (cond ...)
  (print error message ...))
```